pkg/util/net_test.go

```diff
@@ -0,0 +1,84 @@
+package util
+
+import (
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Thanks for the unit test, but please follow the guidelines for unit tests https://github.com/k3s-io/k3s/blob/master/tests/TESTING.md#framework. You don't need to rewrite much, just the assertions. We don't use the testify assert package on purpose, we use regular comparisons or DeepEqual with an error. See https://github.com/k3s-io/k3s/blob/master/pkg/configfilearg/parser_test.go#L335 for an example.

Reply:
@dereknola Sorry about that, didn't pay enough attention. Changed the code to be conformant with the guidelines.